logexp/README.md

```diff
@@ -0,0 +1,40 @@
+# LogExp
+
+LogExp is a tool that runs post-test on our kubernetes test clusters.
+It does the job of computing the set of logfiles to be exported (based on the
+node type (master/node), cloud provider, and the node's system services),and
+then actually exports them to the GCS path provided to it.
+
+The following flags need to be supplied to the tool:
+* --node-name (Name of the node this log exporter is running on)
+* --gcs-artifacts-path (Path to the GCS artifacts dir under which to upload logs, for eg: gs://my-logs-bucket/logs)
+* --kubernetes-provider (Cloud provider for this node (gce/gke/aws/kubemark/..))
+* --gcloud-auth-file-path  (Path to gcloud service account file, for authenticating gsutil to write to GCS bucket. Defaults to /etc/service-account/service-account.json)
+* --enable-hollow-node-logs (Enable uploading hollow node logs too. Relevant only for kubemark nodes. Defaults to false)
+* --sleep-duration (Duration to sleep before exiting with success. Useful for making pods schedule with hard anti-affinity when run as a job on a k8s cluster)
+
+## How to run the tool?
+
+Before running the tool on any node, create a secret with the gcloud service account credentials:
+1. Prepare service-account.json file (that should have write access to specified GCS path)
+2. `kubectl create secret generic logexp-service-account --from-file=service-account.json=/path/to/service-account.json`
+
+To run the tool as a pod on a single node:
```

Comment:
I kept it just for quick testing on a single node rather than doing on whole cluster. Also it'll be handy if someone is interested in just a single node's logs instead of the entire cluster on which he might be running an e2e test. Is that a reasonable use or should I remove it?